### DISCUSSION TITLE
unable to access cerficate after changes to microsoft account

### DISCUSSION THREAD

**Discussion by ghost** [rcl-ssl/docs#37] [Q&A] [answered]
Hi, so, about a month ago I changed the user principal name of my work account because new company policies required it to be something like department@domain instead of name@domain. The problem is, now that the certificate has expired it is no longer getting provisioned to my work account (the one with the new name), but to a personal account (that I didn't even know existed) with the old username. When I open the certificates page with my work account I get the following errors: "ERROR - A subscription associated with this id is already active." and "Please subscribe before you use this feature.". The auto-renew feature is no longer working, and the companies' website is throwing a certificate error. What do i do?

**Comment by rcladmin** [comment]
What is the old username?

**Reply by ghost** [reply]
old username is juanes@rma.com.co and new username is soporte@rma.com.co

**Comment by rcladmin** [comment, marked as answer]
can you send an email to support@rclapp.com using both email accounts, just to verify that you own both emails. I can then try to switch the emails on your account.

**Reply by rcladmin** [reply]
Just switched the account , can you login to the RCL Portal and tell me if you can access your account?

**Reply by ghost** [reply]
yes, i can now access the account, i will still send the emails as proof.

**Reply by rcladmin** [reply]
Ok, you must also ensure that the new account that you are logging in with has the necessary access control permission for the subscription containing the app service in order to renew the certificate.

https://docs.rclapp.com/authorization/access-control-user.html

**Reply by ghost** [reply]
oh yeah, i have the global administrator role. I just updated the certificate and the company's website is back up and running, thank you very much for your help

**Reply by rcladmin** [reply]
OK, if you need further assistance please please post here.
